doc/radosgw/s3select.rst

```diff
@@ -32,30 +32,29 @@ the better the performance of s3select**.
 Basic Workflow
 --------------
     
-S3-select query is sent to RGW via `AWS-CLI
+S3 Select queries are sent to RGW via `AWS-CLI
 <https://docs.aws.amazon.com/cli/latest/reference/s3api/select-object-content.html>`_
 
-It passes the authentication and permission process as an incoming message
-(POST). **RGWSelectObj_ObjStore_S3::send_response_data** is the “entry point”,
-it handles each fetched chunk according to input object-key.
-**send_response_data** is first handling the input query, it extracts the query
-and other CLI parameters.
+S3 Select passes the authentication and permission process as an incoming
+message (POST). ``RGWSelectObj_ObjStore_S3::send_response_data`` is the entry
```

Comment:
I'm not sure what this first sentence is saying.  It passes the process? Does that mean a negotiation of protocols? Or should s/process/parameters/?

Reply:
I think it's probably "parameters".

Reply:
yes. my mistake. 
it should be parameters.
Tx.

Reply:
It's updated now.